compiler/compile/ResolvedMethod.hpp

```diff
@@ -42,6 +42,7 @@ class TR_TraceFragment;
 namespace TR { class CodeGenerator; }
 namespace TR { class Compilation; }
 namespace TR { class IlGeneratorMethodDetails; }
+namespace TR { class IlType; }
```

Comment:
I think this shouldn't be needed anymore.

Reply:
Fixed.  Rebased and repushed.